### PR TITLE
HPCC-27179 Start-all should configure elastic4hpcclogs logaccess

### PR DESCRIPTION
**Pull request by rpastrana**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by rpastrana** [comment]
This PR will be rebased once [HPCC-27167 Add wslogaccess to containerized eclwatch](https://github.com/hpcc-systems/HPCC-Platform/pull/15766/commits/ffdb961b0bec8c182a7df001e9a40f742fc38c6c)
[ffdb961](https://github.com/hpcc-systems/HPCC-Platform/pull/15766/commits/ffdb961b0bec8c182a7df001e9a40f742fc38c6c) is merged.

**Comment by richardkchapman** [comment]
@rpastrana Looks ok, but will need rebasing to remove the first commit. I have merged the branch you refer to I think

**Comment by rpastrana** [comment]
@g-pan please review README

File: helm/examples/logging/README.md [hunk 1]
```diff
@@ -20,6 +20,43 @@ Cluster-level logging for the containerized HPCC Systems cluster can be accompli
 
 _Specific logging tool examples can be found in the child folders_
 
+## HPCC Systems log access
+
+Oout of the box, HPCC component logs are handled by K8s drivers, and exposed via the kubectl logs pod command.
```
Comment (g-pan) [inline helm/examples/logging/README.md:25]: Out

File: helm/examples/logging/README.md [hunk 2]
```diff
@@ -20,6 +20,43 @@ Cluster-level logging for the containerized HPCC Systems cluster can be accompli
...
+Access to logs processed by other processes such as Elastic Stack, Azure log analytics, etc. is provided by the chosen log processor solution.
+However, HPCC provides a standard interface into the logs. To enable this feature, information about the chosent log processor solution must be provided in
```
Comment (g-pan) [inline helm/examples/logging/README.md:27]: chosen log processor

**Review by g-pan — COMMENTED** [review]
see comments in line

**Review by g-pan — APPROVED** [review]

**Comment by rpastrana** [comment]
@richardkchapman rebased, and README reviewed.

**Pull request merged**